symbols/overlay29.yml

```diff
@@ -500,7 +526,7 @@ overlay29:
         First it calls TrySpawnMonsterAndTickSpawnCounter, then tries to activate the Plus and Minus abilities for both allies and enemies, and finally calls TryForcedLoss.
         
         No params.
-    - name: FloorIsOver
+    - name: IsFloorOver
```

Comment:
Honestly I'm terrible at staying consistent about this 😅. No objections on this change though.